Prune more Toit v1 content (#389)
toitware/web-docs: docs/support/troubleshoot/vscodetroubleshoot.mdx

**Error: "Couldn't start client Language Server"**

> Make sure `jag` is in your path. When you [install](../../getstarted/device) the Jaguar executable make sure to

Reviewer: ```suggestion
Make sure `jag` is in your path. When you [install](../../getstarted/device) the Jaguar executable
```

> check "Add the Jaguar CLI application directory to the PATH variable" when running the installer.

Reviewer: ```suggestion
check the "Add the Jaguar CLI application directory to the PATH variable" box when running the installer.
```

Restart VS Code (reloading the window is not enough).

> To check whether `jag` is found in VS Code, open a terminal in VS Code (Ctrl+`) and run

Reviewer: ```suggestion
To verify that `jag` is found in VS Code, open a terminal in VS Code (Ctrl+`) and run
```
